Docs/neurons.md

```diff
@@ -161,5 +161,20 @@ neurons:
 
 Here, the first neuron will use the default tts as set in the settings.yml file. The second neuron will use the tts "acapela".
```

Comment:
It's just that the doc talk about accapela and your example is based on Pico. So just update this sentence with pico to be consistent.